### PR TITLE
AcronisInternationalGmbH.AcronisCyberProtectHomeOffice version 26.1.1.39703

### PR DESCRIPTION
**Pull request by Rolling2405**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36274)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: AcronisInternationalGmbH.AcronisCyberProtectHomeOffice
```
Comment (Trenly) [inline manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.installer.yaml:4]:
```suggestion
PackageIdentifier: Acronis.CyberProtectHomeOffice
```

File: manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: AcronisInternationalGmbH.AcronisCyberProtectHomeOffice
```
Comment (Trenly) [inline manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Acronis.CyberProtectHomeOffice
```

File: manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: AcronisInternationalGmbH.AcronisCyberProtectHomeOffice
```
Comment (Trenly) [inline manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.yaml:4]:
```suggestion
PackageIdentifier: Acronis.CyberProtectHomeOffice
```

File: manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: AcronisInternationalGmbH.AcronisCyberProtectHomeOffice
+PackageVersion: 26.1.1.39703
+PackageLocale: en-US
+Publisher: Acronis International GmbH
```
Comment (Trenly) [inline manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.locale.en-US.yaml:7]:
```suggestion
Publisher: Acronis
```

File: manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: AcronisInternationalGmbH.AcronisCyberProtectHomeOffice
+PackageVersion: 26.1.1.39703
```
Comment (Trenly) [inline manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.installer.yaml:5]:
```suggestion
PackageVersion: 26.1.39703
```

File: manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: AcronisInternationalGmbH.AcronisCyberProtectHomeOffice
+PackageVersion: 26.1.1.39703
```
Comment (Trenly) [inline manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 26.1.39703
```

File: manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: AcronisInternationalGmbH.AcronisCyberProtectHomeOffice
+PackageVersion: 26.1.1.39703
```
Comment (Trenly) [inline manifests/a/AcronisInternationalGmbH/AcronisCyberProtectHomeOffice/26.1.1.39703/AcronisInternationalGmbH.AcronisCyberProtectHomeOffice.yaml:5]:
```suggestion
PackageVersion: 26.1.39703
```

**Review by Trenly — CHANGES_REQUESTED** [review]
In addition to https://github.com/microsoft/winget-pkgs/pull/36274#pullrequestreview-817233338
The Publisher and version will need to match what is in the ARP entry -
image

**Pull request closed without merging**